Intersect.Editor/Localization/Strings.cs

```diff
@@ -2085,7 +2094,8 @@ public struct EventConditional
                 {17, @"Item Equipped is..."},
                 {18, @"Has X free Inventory slots..." },
                 {19, @"In Guild With At Least Rank..." },
-                {20, @"Map Zone Type is..." }
+                {20, @"Map Zone Type is..." },
+                {21, @"Check Equipped Slot... " }
```

Comment:
Delete the extra space ` ` in the string, and end the line with `,` so that future additions don't need to modify two lines.